changelog/7938.improvement.rst

```diff
@@ -0,0 +1 @@
+Stepwise plugin improvements and a new ``--sw-skip`` shorthand argument is now available
```

Comment:
"Stepwise plugin improvements" doesn't say much to a reader of the changelog, so I'd omit this part.

I would mention that `--sw-skip` is a shorthand for `--stepwise-skip`.